src/browser/BrowserService.cpp

```diff
@@ -1108,18 +1113,24 @@ bool BrowserService::removeFirstDomain(QString& hostname)
 
 /* Test if a search URL matches a custom entry. If the URL has the schema "keepassxc", some special checks will be made.
  * Otherwise, this simply delegates to handleURL(). */
-bool BrowserService::handleEntry(Entry* entry, const QString& url, const QString& submitUrl)
+bool BrowserService::handleEntry(Entry* entry,
+                                 const QString& url,
+                                 const QString& submitUrl,
+                                 const bool omitWwwSubdomain)
 {
     // Use this special scheme to find entries by UUID
     if (url.startsWith("keepassxc://by-uuid/")) {
         return url.endsWith("by-uuid/" + entry->uuidToHex());
     } else if (url.startsWith("keepassxc://by-path/")) {
         return url.endsWith("by-path/" + entry->path());
     }
-    return handleURL(entry->url(), url, submitUrl);
+    return handleURL(entry->url(), url, submitUrl, omitWwwSubdomain);
```

Comment:
Shouldn't we be checking all the URL's here? `Entry::getAllUrls()`

Reply:
Done.